stratum/src/lib.rs

```diff
@@ -179,6 +182,32 @@ impl PushWorkHandler for Stratum {
 	}
 }
 
+lazy_static! {
+	static ref LOG_DUMMY: bool = {
```

Comment:
I think either this or the `lazy_static!` invocation can be marked as `#[cfg(test)]` also